docs/source/conf.py

```diff
@@ -49,6 +49,12 @@
 
 # napoleon autodoc config
 napoleon_include_init_with_doc = True
+autodoc_mock_imports = [
```

Comment:
everything inslide `lib/python/` that is `imported` and not already needed in `docs/requirements.txt`